docs/ref/models/expressions.txt

```diff
@@ -450,6 +450,10 @@ instantiating the model field as any arguments relating to data validation
 (``max_length``, ``max_digits``, etc.) will not be enforced on the expression's
 output value.
 
+Django also uses ``output_field`` to send the value to the database with the
```

Comment:
I don't understand this note. Is it related to the patch?

Reply:
I meant to say you need to specify `output_field` if a simple `Value(v)` call doesn't serialize `v` in the correct format. For example, you could mean to send `'[1,2]'::json` or `'[1,2]'::int[]`.